### ISSUE TITLE
jekyllMaps is not defined

### ISSUE THREAD

**Issue by wharkcom** [ayastreb/jekyll-maps#31]
When I add {% google_map %} to my page, the following is output in the final HTML:
```
<div id="1fc4ca6d-5e80-4134-8472-5516b81a3d40" style="width:600px;height:400px;"></div>
<script `type="text/javascript">
jekyllMaps.register('1fc4ca6d-5e80-4134-8472-5516b81a3d40', [{ "latitude": 32.7661546, "longitude": -61.9589893, "title": "Current Location", "url": "/current-location/", "image": "" }], { "baseUrl": "/", "useCluster": true, "showMarker": true, "showMarkerPopup": true });
</script>
```
This ends up throwing the console error "jekyllMaps is not defined".

**Comment by ayastreb** [comment]
Hi! Which version of Jekyll and jekyll-maps do you use?

jekyll-maps should also inject JS code in the `<head>` of the page.

**Comment by wharkcom** [comment]
jekyll (3.7.0) and jekyll-maps (2.1.0).

I don't see a related script in the `<head>`.

**Comment by ayastreb** [comment]
Do you have a source code available somewhere?

**Comment by wharkcom** [comment]
https://github.com/wharkcom/sleek
https://github.com/wharkcom/sleek/blob/master/_pages/current-location.md

I did a better search, and the script does end up in the head, but not in the right place. Originally, it ended up within my `<title>` tag, which was generated by jekyll-seo-tag. I disabled that, and now the script ends up within the `<script>` tag for google tag manager.

```
<script async="" src="https://www.googletagmanager.com/gtag/js?id=GTM-5JDH493">
	<script type='text/javascript'>
	  /* global google */
	/* global MarkerClusterer */
	// eslint-disable-next-line no-unused-vars
	var jekyllMaps = (function() {
	  'use strict'
	  var clusterSettings = {}
```

**Comment by ayastreb** [comment]
Should be fixed in 2.1.1 😄

**Comment by wharkcom** [comment]
Thanks! It's working now.
